debian/ceph-base.install

```diff
@@ -11,6 +11,7 @@ usr/lib/ceph/erasure-code/*
 usr/lib/rados-classes/*
 usr/share/doc/ceph/sample.ceph.conf
 usr/share/doc/ceph/sample.fetch_config
+usr/share/doc/ceph/COPYING
```

Comment:
as discussed over IRC, i don't think we need two copies of copyright / COPYING in debian packaging.